components/odh-notebook-controller/controllers/notebook_controller_test.go

```diff
@@ -605,7 +605,6 @@ var _ = Describe("The Openshift Notebook controller", func() {
 					"notebooks.opendatahub.io/inject-oauth":     "true",
 					"notebooks.opendatahub.io/foo":              "bar",
 					"notebooks.opendatahub.io/oauth-logout-url": "https://example.notebook-url/notebook/" + Namespace + "/" + Name,
-					"kubeflow-resource-stopped":                 "odh-notebook-controller-lock",
```

Comment:
this change worries me; the lock annotation is a weird hack and I'm uneasy to touch it